assignments/prototypes.js

```diff
@@ -8,40 +8,64 @@
   Each constructor function has unique properites and methods that are defined in their block comments below:
 */
   
-/*
-  === GameObject ===
-  * createdAt
-  * dimensions
-  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
-*/
 
-/*
-  === CharacterStats ===
-  * hp
-  * name
-  * takeDamage() // prototype method -> returns the string '<object name> took damage.'
-  * should inherit destroy() from GameObject's prototype
-*/
+  // === GameObject ===
+  // * createdAt
+  // * dimensions
+  // * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+  
+  function GameObject(gameAttr) {
+    this.createdAt = gameAttr.createAt;
+    this.dimensions = gameAttr.dimensions;
+    this.destroy = function (){
+      return `${this.object} was removed from the game.`;
+    }
+  }
+
+  // === CharacterStats ===
+  // * hp
+  // * name
+  // * takeDamage() // prototype method -> returns the string '<object name> took damage.'
+  // * should inherit destroy() from GameObject's prototype
+
+  function CharacterStats(charAttr) {
+    // GameObject.call(this, charAttr);
```

Comment:
Why is this line commented out?

Reply:
After putting

>GameObject.call(this, charAttr);

back in, add in the Object.create() statement as well. If you're not sure how to do that, review the lecture video again.